src/uu/install/src/install.rs

```diff
@@ -391,21 +391,44 @@ fn behavior(matches: &ArgMatches) -> UResult<Behavior> {
         show_error!("Options --compare and --strip are mutually exclusive");
         return Err(1.into());
     }
-    Ok(Behavior {
-        main_function,
-        specified_mode,
-        backup_mode,
-        suffix: backup_control::determine_backup_suffix(matches),
-        owner: matches
+
+    let owner = matches
             .get_one::<String>(OPT_OWNER)
             .map(|s| s.as_str())
             .unwrap_or("")
-            .to_string(),
-        group: matches
+            .to_string();
+
+    let owner_id = if !owner.is_empty() {
+        match usr2uid(&owner) {
+            Ok(u) => Some(u),
+            _ => return Err(InstallError::InvalidUser(owner.clone()).into()),
+        }
+    } else {
+        None
+    };
+
+    let group = matches
             .get_one::<String>(OPT_GROUP)
             .map(|s| s.as_str())
             .unwrap_or("")
-            .to_string(),
+            .to_string();
+
+    let group_id = if !group.is_empty() {
+        match grp2gid(&group) {
+            Ok(g) => Some(g),
+            _ => return Err(InstallError::InvalidGroup(group.clone()).into()),
```

Comment:
Nit: let's be explicit about `Err(_)` here, instead of `_` (and also in the other match)

Reply:
Agreed, updated both to be more explicit.  Originally left it as it was moved from the original section.